docs/source/installation.rst

```diff
@@ -2,152 +2,125 @@ Installation
 ============
 
 Supported OS
-------------
+++++++++++++
 
-Currently, we only support ``MacOS`` and ``Linux`` operating systems. ``Windows``
-users have the possibility to install and use ``ivadomed`` via
-`Windows Subsystem for Linux (WSL) <https://docs.microsoft.com/en-us/windows/wsl/>`_. The steps below (about updating bashrc) are strongly recommended for MacOS users in the installation process but are optional for Linux users.
+Currently, ``ivadomed`` only supports GPU/CPU on ``Linux`` and ``Windows`` and CPU only on ``macOS`` and `Windows Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/>`_.
 
-Open your bash/zsh script file with editor on your computer.
-
-    If you are using bash shell
-    ::
-
-        vim ~/.bashrc
+Step 1: Setup dedicated python environment
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-    If you are using zsh shell
-    ::
-        
-        vim ~/.zshrc
+You can setup ``ivadomed`` using either Conda or Venv:
 
-Write in your .bashrc/.zshrc file with following line.
+.. tabs::
 
-::
+    .. tab:: Install via ``conda``
 
-    export HDF5_USE_FILE_LOCKING='FALSE'
+        This is the easiest way for personal computers.
 
-Save this change and restart your terminal to apply the change.
+        1. Create new conda environment using ``environment.yml`` file
 
-There are fundamentally two different approaches to install ``IvadoMed``:
+        ::
 
-1) Install via Conda
-    This is the easiest way for personal computers.
+            conda env create --name ivadomed_env
 
-2) Install via Venv/VirtualEnv
-    This is compatible with ComputeCanada cluster environment.
+        2. Activate the new conda environment (default named ``ivadomed_env``)
 
-Approach 1: Conda
-------------------
+        ::
 
-Step 1: Clone the `ivadomed <https://github.com/ivadomed/ivadomed>`_ repository.
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+            conda activate ivadomed_env
 
-::
+    .. tab:: Install via ``venv``
 
-    git clone https://github.com/neuropoly/ivadomed.git
-    cd ivadomed
+        1. Setup Python Venv Virtual Environment.
 
-Step 2: Create new Conda Env called IvadoMedEnv (may take 5 to 15 minutes)
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        ``ivadomed`` requires Python >= 3.6 and <3.10. We recommend working under a virtual environment, which could be set as follows:
 
-::
+        ::
 
-    conda env create --file environment.yml
+            python -m venv ivadomed_env
```

Comment:
This did not work for me, I had to `sudo apt install python3.8-venv` first, and then `python3.8 -m venv ivadomed_env`.

Reply:
Also, there was a previous discussion about being unable to set the python is `venv`. I think this is problematic, and as a user, I want to be able to choose my version. I think the way to do this is maybe with:
`python3.X -m venv ivadomed_env`

Reply:
Ah! Very good point! `python` runs the risk of mapping to `python2`, but if we tell the user to be explicit about their python version, then we avoid this issue.

We could say something like:

```rest
First, make sure that a compatible version of Python 3 is installed on your system by running:

.. code::

   python3 --version

If your system's Python is not 3.7, 3.8, or 3.9 (or if you don't have Python 3 installed at all), please [install Python](https://realpython.com/installing-python/) before continuing.

Once you have a supported version of Python installed, run the following command (where `python3.X` corresponds to the installed Python version)

.. code::

   python3.X -m venv ivadomed_env
```

Reply:
Very good point. I didn't consider this. Added. :)